src/game_picture.cpp

```diff
@@ -167,13 +223,28 @@ void Game_Picture::RequestPictureSprite() {
 void Game_Picture::OnPictureSpriteReady(FileRequestResult*) {
 	RPG::SavePicture& data = GetData();
 
-	BitmapRef bitmap = Cache::Picture(data.name, data.transparency);
+	whole_bitmap = Cache::Picture(data.name, data.transparency);
 
 	sprite.reset(new Sprite());
-	sprite->SetBitmap(bitmap);
+	sprite->SetBitmap(whole_bitmap);
+
 	UpdateSprite();
 }
 
+bool Game_Picture::HasSpritesheet() const {
+	RPG::SavePicture& data = GetData();
+
+	if (data.spritesheet_rows < 1 || data.spritesheet_cols < 1) {
+		return false;
+	}
+
+	if (data.spritesheet_rows == 1 && data.spritesheet_cols == 1) {
+		return false;
+	}
+
+	return true;
+}
```

Comment:
Why not simpler?

```cpp
if (data.spritesheet_rows > 1 || data.spritesheet_cols > 1)
	return true;

return false;
```

Reply:
Good find 👍

Reply:
Simplified the check a bit.
But have to keep the first if-check because it's possible that only one of the values is > 1 due to uninitialized arguments.